packages/wallet/package.json

```diff
@@ -55,7 +56,7 @@
     "@fuel-ts/transactions": "workspace:*",
     "@fuel-ts/utils": "workspace:*",
     "@fuels/vm-asm": "0.34.1",
-    "@fuel-ts/crypto": "workspace:*",
+    "tree-kill": "^1.2.2",
```

Comment:
You need to add `@fuel-ts/fuel-core` as a dependency here to guarantee that it will work even outside the mono repo or in case users install the `wallet` package in isolation.